src/capi.rs

```diff
@@ -50,7 +50,7 @@ impl From<Arc<rav1e::Frame<u16>>> for FrameInternal {
 
 /// Raw video Frame
 ///
-/// It can be allocated through rav1e_frame_new(), populated using rav1e_frame_fill_plane()
+/// It can be allocated throught rav1e_frame_new(), populated using rav1e_frame_fill_plane()
```

Comment:
That change is incorrect, please revert it.

Reply:
Done !